app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,48 @@
  *= require foundation_and_overrides
  */
 
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+
  /************** BASIC STYLING *****************************/
 
 #content {
-  background-color: #DADADA;
+  background-color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`